src/components/Proposals/ProposalPage/ShareVoteDialog/TenantLogo.tsx

```diff
@@ -1,577 +1,114 @@
 import { TENANT_NAMESPACES } from "@/lib/constants";
 import { TenantNamespace } from "@/lib/types";
-
+import optimismLogo from "@/assets/tenant/optimism_share_logo.svg";
+import scrollLogo from "@/assets/tenant/scroll_logo.svg";
+import deriveLogo from "@/assets/tenant/derive_logo.svg";
+import uniswapLogo from "@/assets/tenant/uniswap_logo.svg";
+import xaiLogo from "@/assets/tenant/xai_logo.svg";
+import etherfiLogo from "@/assets/tenant/etherfi_logo.svg";
+import ensLogo from "@/assets/tenant/ens_logo.svg";
+import cyberLogo from "@/assets/tenant/cyber_logo.svg";
+import boostLogo from "@/assets/tenant/boost_logo.svg";
+import b3Logo from "@/assets/tenant/b3_logo.svg";
+import pguildLogo from "@/assets/tenant/pguild_logo.svg";
+import Image from "next/image";
 export const ogLogoForShareVote = (namespace: TenantNamespace) => {
-  switch (namespace) {
+  const normalizedNamespace = namespace?.toLowerCase();
+  switch (normalizedNamespace) {
     case TENANT_NAMESPACES.SCROLL:
       return (
-        <svg
-          xmlns="http://www.w3.org/2000/svg"
-          xmlnsXlink="http://www.w3.org/1999/xlink"
-          x="0px"
-          y="0px"
-          width="52"
-          height="52"
+        <Image
+          src={scrollLogo}
```

Comment:
Simplified this a bit. Was hoping to figure out a pattern for OGs across but since server side rendering is edge run time, makes things a bit more complicated.